src/app/not-found.tsx

```diff
@@ -1,13 +1,23 @@
-import Link from "next/link";
+"use client";
 import React from "react";
+import {
+    ErrorCenterer,
+    ErrorPanel,
+    ErrorLargeText,
+    ErrorSecondaryText,
+} from "@/app/errorPageStyling";
+import LinkButton from "@/components/Buttons/LinkButton";
 
-const NotFound: React.FC<{}> = () => {
+const Error: React.FC<{}> = () => {
     return (
-        <>
-            <p> No page found. </p>
-            <Link href="/clients"> Home </Link>
-        </>
+        <ErrorCenterer>
+            <ErrorPanel elevation={5}>
+                <ErrorLargeText>OOPS!</ErrorLargeText>
+                <ErrorSecondaryText>404 Error || This page does not exist</ErrorSecondaryText>
```

Comment:
Maybe just `This page does not exist` ? To make it less "techy" for general users

Reply:
404 Error is common on error pages, if you google page not found error page that is usually what comes up!

Reply:
I kinda like the two lines ngl :/